vanillajs/cards/app.js

```diff
@@ -0,0 +1,25 @@
+const placeholderInfo = {
+  image: "https://via.placeholder.com/150/771796",
+  name: "Jane Doe",
+  title: "VP Engineering"
+};
+
+class Card {
+  constructor(container) {
+    this.container = document.querySelector(container);
+  }
+
+  init() {
```

Comment:
You're taking the global placeholderInfo, instead of passing it to the constructor. How would you create more cards with different data?

Reply:
passed dataObj as a parameter so it can be grabbed from a fetch